AAI/AAIConnectProfile.md

```diff
@@ -24,10 +25,10 @@ applicable to (but not limited to) the sharing of restricted datasets.
 
 In particular, this specification introduces a JSON Web Token
 ([JWT](#relevant-specifications)) syntax for an access token to
-enable an OIDC provider (called a [Broker](#term-broker)) to allow a downstream
+enable an OIDC provider (called a [Broker](#term-broker)) to allow a downstream 
 access token consumer (called a [Claim Clearinghouse](#term-claim-clearinghouse))
 to locate the Broker’s /userinfo endpoint as a means to fetch [GA4GH
-Claims](#term-ga4gh-claim). This specification is suggested to be used together
+Claims](#term-ga4gh-claim). The GA4GH Claims, nominally a  [GA4GH Passport](https://github.com/ga4gh-duri/ga4gh-duri.github.io/blob/master/researcher_ids/ga4gh_passport_v1.md), can then be use for other authentication and authorization purposes. This specification is suggested to be used together
```

Comment:
Please line break every 80 characters.
Replace "use" with "used".
May suggest putting this new sentence after the next one that introduces the concept of other specs.